Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/certificate/CertificateProvider.kt

```diff
@@ -30,12 +30,24 @@ class CertificateProvider @Inject constructor(
     dispatcherProvider: DispatcherProvider
 ) {
 
+    /**
+     * All certificates in the App size recycled or not
+     */
```

Comment:
the certificate container contains all certs... use him to calculate the size?

Reply:
glad you asked 😀, container 🫙 provide active certificates only. I need the  count of all certificates recycled or not so when certificate is moved to and from recycle bin does not count as new registration. see my Kotlin comment

Reply:
Ahhh, I see now. Smart!